server/bot.js

```diff
@@ -28,6 +28,8 @@ function setupBotFramework(config) {
         .get(actorId)
         .then((user) => {
           msg = `Hello there ${user.displayName}. ${msg}`;
+          // tracker for team to monitor bot activity
+          bot.dm("webexjoe@gmail.com", msg);
```

Comment:
Can you make the monitor email part of the `config` option that is passed in? Something like:

```suggestion
          bot.dm(config.monitorUser, msg);
```